src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -64,10 +64,10 @@
      * Values should be annotated with the feature's {@code JEP}.
      */
     public enum Feature {
-        // not used
+        // not used, but required for interim javac to not warn.
         VIRTUAL_THREADS,
-        @JEP(number=442, title="Foreign Function & Memory API", status="Third Preview")
         FOREIGN,
```

Comment:
Note that I had to leave the enum constant here for now, since otherwise we encounter a warning when building interim javac.

```
jorn@Jorn-PC:/mnt/h/openjdk/foreign-abi$ make run-test-jdk_foreign CONF=windows-fastdebug
Using exact match for CONF=windows-fastdebug (other matches are possible)
Building target 'run-test-jdk_foreign' in configuration 'windows-fastdebug'
warning: unknown enum constant Feature.FOREIGN
error: warnings found and -Werror specified
1 error
1 warning
gmake[3]: *** [CompileInterimLangtools.gmk:127: /mnt/h/openjdk/foreign-abi/build/windows-fastdebug/buildtools/interim_langtools_modules/java.compiler.interim/_the.BUILD_java.compiler.interim_batch] Error 1
gmake[2]: *** [make/Main.gmk:78: interim-langtools] Error 2
gmake[2]: *** Waiting for unfinished jobs....

ERROR: Build failed for target 'run-test-jdk_foreign' in configuration 'windows-fastdebug' (exit code 2)

=== Output from failing command(s) repeated here ===
* For target buildtools_interim_langtools_modules_java.compiler.interim__the.BUILD_java.compiler.interim_batch:
warning: unknown enum constant Feature.FOREIGN
error: warnings found and -Werror specified
1 error
1 warning

* All command lines available in /mnt/h/openjdk/foreign-abi/build/windows-fastdebug/make-support/failure-logs.
=== End of repeated output ===

No indication of failed target found.
HELP: Try searching the build log for '] Error'.
HELP: Run 'make doctor' to diagnose build problems.

make[1]: *** [/mnt/h/openjdk/foreign-abi/make/Init.gmk:323: main] Error 2
make: *** [/mnt/h/openjdk/foreign-abi/make/Init.gmk:189: run-test-jdk_foreign] Error 2
```